### DISCUSSION TITLE
"Invalid Authentication"

### DISCUSSION THREAD

**Discussion by rohan3au** [alandtse/tesla#109] [General]
Getting this error when trying to configure the integration. Using the Auth for Tesla iOS app.

Any ideas?

**Comment by alandtse** [comment]
Are you based in [China](https://github.com/alandtse/tesla/issues/36#issuecomment-977616909)? Or you're pasting the wrong token.

**Reply by rohan3au** [reply]
Not based in China, and have tried various times and confirmed I'm inputting the correct token.

**Reply by KivinChiu** [reply]
我在中国，我也遇到了同样的问题，请问如何解决？
